todo_app/app.py

```diff
@@ -45,9 +82,8 @@ def unauthenticated():
 
     @login_manager.user_loader
     def load_user(user_id):
-        print('load_user', user_id)
+        app.logger.debug('load_user', user_id)
```

Comment:
This errors when I hit it - if we want the log to format this correctly I think we need to provide some guidance, e.g. either tell them where/how to add the arguments in:
```
app.logger.debug('user_info %s', user_info)
```
Or handle it ourselves:
```
app.logger.debug(f'user_info: {user_info}')
```